hack/bundler.sh

```diff
@@ -2,4 +2,4 @@
 set -euxo pipefail
 cd route-monitor-operator
 make packagemanifests
-chmod 775 -R packagemanifests
+# chmod 775 -R $BUNDLE_DIR
```

Comment:
That's going to bork the jenkins slave again

Reply:
yes but I'm not yet sure how to get it right...

Reply:
because it will change permissions of existing files. Not sure if that would even be a problem